cloudbuild.yaml

```diff
@@ -14,6 +14,7 @@ substitutions:
   _MYSQL_TAG: "5.7"
   _MYSQL_ROOT_PASSWORD: ""
   _MYSQL_PASSWORD: ""
+  _CODECOV_TOKEN: "" # The auth token for uploading coverage to Codecov.
```

Comment:
You probably know this already, but the `sekret` value can be added in the cloudbuild trigger configuration in the GCP console.

Reply:
Yep, that's what I did.